TypeScript/README.md

```diff
@@ -11,7 +11,7 @@
 npm install --save blurhash
 ```
 
-See [react-blurhash](https://github.com/woltapp/react-blurhash) to use blurhash with React.
+See [react-blurhash](https://github.com/woltapp/react-blurhash) to use blurhash with React or [stimulus-blurhash](https://github.com/bb/stimulus-blurhash) to use it with Hotwire Stimulus.
```

Comment:
I think this implementation should be listed under implementations in README!

Reply:
I can of course add both react-blurhash and stimulus-blurhash to the global README. However, neither of those are actual implementations. They both have the TypeScript implementation as a dependency. So I'd add both react-blurhash and stimulus-blurhash as sub-bullets of the TypeScript implementation? Or would you prefer somewhere else?